cli/azd/internal/appdetect/java.go

```diff
@@ -177,6 +177,13 @@ func detectDependencies(mavenProject *mavenProject, project *Project) (*Project,
 			databaseDepMap[DbMongo] = struct{}{}
 		}
 
+		// we need to figure out multiple projects are using the same service bus
+		if dep.GroupId == "com.azure.spring" && dep.ArtifactId == "spring-cloud-azure-starter-servicebus-jms" {
+			project.AzureDeps = append(project.AzureDeps, AzureDepServiceBus{
+				IsJms: true,
```

Comment:
How about adding `IsJms: false` after line 195? Not relying on default value will make code easier to read.